Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,88 @@
+// DATA TYPES
+// 1. Declare a variable on each type using two options: plain creation, creation with constructor.
+// 2. Use operator typeof on each of them.
+//number
+let plainNumber = 10;
```

Comment:
always use `const` if you are not reassigning the variable

Reply:
Thank you. Got it and changed in the file.